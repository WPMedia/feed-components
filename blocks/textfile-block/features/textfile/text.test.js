import TextFile from './text'

const simple =
  'User-agent: *\nAllow: /\n\nSitemap: http://www.example.com/sitemap.xml'

it('should render the simple text', () => {
  const textFile = TextFile({
    customFields: {
      Text: simple,
    },
  })
  expect(textFile).toMatchSnapshot()
})

it('should not render anything when no data is given', () => {
  const textFile = TextFile({})
  expect(textFile).toMatchSnapshot()
})
