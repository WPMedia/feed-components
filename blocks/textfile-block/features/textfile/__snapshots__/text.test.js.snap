// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should not render anything when no data is given 1`] = `""`;

exports[`should render the simple text 1`] = `
"User-agent: *
Allow: /

Sitemap: http://www.example.com/sitemap.xml"
`;
